admin.rb

```diff
@@ -12,20 +12,60 @@
 #if you user is signed in but they are not an admin, show "you can see all the pages!"
 #if the user is not signed in, show "you can't see any of the pages, please sign in!"
 
+def user_permission
+
 #ask the user if they pay their bills (yes/no)
 
+puts("Are your bills paid?")
```

Comment:
It's cool to see that you did this differently: didn't use parameters and simply called for the user input in the function - nice!